core/src/main/kotlin/api/OrganizationsRoute.kt

```diff
@@ -263,16 +265,14 @@ fun Route.organizations() = route("organizations") {
             }
         }
 
-        route("groups") {
-            // Instead of identifying arbitrary groups with a groupId, there are only 3 groups with fixed
-            // groupId "readers", "writers" or "admins".
-            route("{groupId}") {
-                put(putUserToOrganizationGroup) {
+        route("roles") {
+            route("{role}") {
+                put(putOrganizationRoleToUser) {
                     requirePermission(OrganizationPermission.MANAGE_GROUPS)
 
                     val user = call.receive<Username>()
                     val organizationId = call.requireIdParameter("organizationId")
-                    val groupId = call.requireParameter("groupId")
+                    val role = call.requireEnumParameter<OrganizationRole>("role").mapToModel()
```

Comment:
Renaming the parameter is a breaking change of the REST API. Is there a way to make this more obvious? At the very least, the commit title should have a "!".